mathesar_ui/src/pages/admin-users/UserRow.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  import type { User } from '@mathesar/api/users';
+  import { Icon } from '@mathesar/component-library';
+  import { iconUser } from '@mathesar/icons';
+  import { getEditUsersPageUrl } from '@mathesar/routes/urls';
+
+  export let user: User;
+
+  // TODO: Update the icon
+  $: userTypeIcon = user.is_superuser ? iconUser : iconUser;
```

Comment:
Is this logic correct?

Reply:
Yeah, this is fine. I don't think we need to update the icon for now, we can use the same icon for admin and normal user.